nilearn/_utils/fixes/scikit_learn_validation.py

```diff
@@ -0,0 +1,400 @@
+# scikit-learn v0.17
+
+import numpy as np
+import warnings
+import scipy.sparse as sp
+
+
+class NotFittedError(ValueError, AttributeError):
+    """Exception class to raise if estimator is used before fitting
+    This class inherits from both ValueError and AttributeError to help with
+    exception handling and backward compatibility.
+    """
+
+
+def check_array(array, accept_sparse=None, dtype="numeric", order=None,
```

Comment:
Good catch. Isn't it the opposite? Isn't `check_x_y` supposed to replace `check_array`?